libs/application/templates/party-application/src/lib/PartyApplicationTemplate.ts

```diff
@@ -26,14 +26,28 @@ enum Roles {
   APPLICANT = 'applicant',
   SIGNATUREE = 'signaturee',
 }
+
+const EndorsementSchema = z.object({
+  id: z.number(),
+  date: z.string(),
+  name: z.string(),
+  nationalId: z.string(),
+  address: z.string(),
+  hasWarning: z.boolean(),
+})
+
 const dataSchema = z.object({
   constituency: z
     .string()
     .refine((v) => v, m.validation.selectConstituency.defaultMessage as string),
   approveDisclaimer: z.boolean().refine((v) => v, {
     message: m.validation.approveTerms.defaultMessage as string,
   }),
+  endorsements: z.array(EndorsementSchema).optional(), //todo: validate
```

Comment:
let's remember to do this validation! i added [this ticket in asana](https://app.asana.com/0/1200022230652074/1200359790125282/f)